STM32/Util/Inc/systemInfo.h

```diff
@@ -43,4 +60,21 @@ int getBoardInfo(BoardType* bdt, SubBoardType* sbdt);
 // Return 0 if data is valid, else negative value.
 int getPcbVersion(pcbVersion* ver);
 
+// Description: set a board status field.
+// @param field: a 1 bit shifted to the field index to be set.
+void bsSetField(uint32_t field);
+
+// Description: set a board status field.
+// @param field: a 1 bit shifted to the field index to be cleared.
+void bsClearField(uint32_t field);
```

Comment:
Is it also helpful to have the following function? To save needing to OR `BS_ERROR_Msk` every time a status change is an error.
```
void bsSetError(uint32_t field) { BS.boardStatus |= BS_ERROR_Msk | field; }
```